pkg/operation/cloudbotanist/localbotanist/localbotanist.go

```diff
@@ -32,7 +32,7 @@ func New(o *operation.Operation) (*LocalBotanist, error) {
 		CloudProviderName: "",
 	}
 	vb.APIServerAddress = *o.Shoot.Info.Spec.DNS.Domain
-	vb.Shoot.InternalClusterDomain = *o.Shoot.Info.Spec.DNS.Domain + ":31443"
+	vb.Shoot.InternalClusterDomain = *o.Shoot.Info.Spec.DNS.Domain
```

Comment:
Not sure why moving the logic of adding the port to the shoot instead of leaving here in the local botanist

Reply:
Valid domain name does not contain port number.